airbyte-cdk/java/airbyte-cdk/db-sources/src/main/kotlin/io/airbyte/cdk/integrations/source/jdbc/AbstractJdbcSource.kt

```diff
@@ -576,6 +576,7 @@ abstract class AbstractJdbcSource<Datatype>(
             }
             .values
             .map { fields: List<JsonNode> -> jsonFieldListToTableInfo(fields) }
+            .filter { ti: TableInfo<CommonField<Datatype>> -> ti.name == tableName }
```

Comment:
the filter suggests that somehow fields belonging to `activity index` get mixed with fields belonging to `activity_index` (with `_`) but it's not clear to me why as the two strings are not equal.

Reply:
this suggests we need to escape as the call translates into `WHERE name LIKE …`